CHANGELOG.rst

```diff
@@ -29,7 +29,7 @@ Forthcoming
 
 0.1.2 (2020-11-24)
 ------------------
-* Add Scanner ports in ScannerConfiguration
+* Add Scanner ports in configuration::ScannerConfiguration
```

Comment:
I think you usually do not change past entries in the changelog.

Reply:
Generally, I like the approach.

Reply:
Minor comments:
- `laserscan.cpp` and `.h` are currently duplicate
- I think the `ScanValidator` in `complete_scan_validator.h` belongs to the protocol_layer
- `format_range` belongs into util namespace/subfolder
- in general all headers should be sorted into the subfolders except maybe `core.h`. What do you think?

Reply:
I think ScanValidator better belongs to MonitoringFrame...

Reply:
Are you sure? This is what the `ScanValidator` does:
```
//! @brief Validates complete scan rounds and detects if MonitoringFrames are missing for a complete scan or
//! if to many MonitoringFrames were received.
```

Reply:
ok